src/DotLiquid.Tests/TemplateTests.cs

```diff
@@ -8,47 +8,56 @@ namespace DotLiquid.Tests
     [TestFixture]
     public class TemplateTests
     {
+
+        private System.Collections.Generic.List<string> TokenizeValidateBackwardCompat(string input)

```

Comment:
Nit nit: spell out words, avoid uncommon abbreviations. i.e. use Compatibility like elsewhere in the code. Yes full English words can get wordy, but modern IDEs offer auto-complete to cut down on the character typing.